Makefile

```diff
@@ -343,7 +343,7 @@ install-helm:
 	./bin/get_helm.sh
 
 create-clientconfig:
-	for ((i = 0; i < $(NUM_CLUSTERS); ++i)); do \
+	for ((i = 1; i < $(NUM_CLUSTERS); ++i)); do \
 		scripts/create-clientconfig.sh \
 		  --namespace $(NS) \
 		  --src-kubeconfig "$(KIND_KUBECONFIG)" \
```

Comment:
Changed here but also in `e2e_framework.go`, it looks like the two are redundant.